.env.default

```diff
@@ -8,8 +8,6 @@ SITE_NAME=Example
 SITE_MAIL=admin@example.com
 ADMIN_NAME=admin
 ADMIN_MAIL=admin@example.com
-ADMIN_PW=admin
```

Comment:
I'd better renamed it to `ADMIN_PASSWORD` **or** allowed drush to generate random one